sdk/tables/azure-data-tables/src/tables_clients.cpp

```diff
@@ -820,6 +820,37 @@ void Models::QueryEntitiesPagedResponse::OnNextPage(const Azure::Core::Context&
   *this = m_tableClient->QueryEntities(m_operationOptions, context);
 }
 
+Azure::Response<Models::TableEntity> TableClient::GetEntity(
+    const std::string& partitionKey,
+    const std::string& rowKey,
+    Core::Context const& context)
+{
+  auto url = m_url;
+  url.AppendPath(m_tableName + "(PartitionKey='" + partitionKey + "',RowKey='" + rowKey + "')");
```

Comment:
Don't you need to URL-encode it? URL encoding would leave most of the characters intact, but would transform, spaces, for example, to `%20`s.